examples/sample_program.py

```diff
@@ -297,7 +297,7 @@ class SampleChef(SushiChef):
         'CHANNEL_DESCRIPTION': 'A sample sushi chef to demo content types.',      # (optional) description of the channel (optional)
     }
 
-    def construct_channel(self, **kwargs):
+    def construct_channel(self, *args, **kwargs):
```

Comment:
Currently, the ricecooker doesn't pass any positional args to to `construct_channel` so it should work even without this, but I guess it's a good idea for scripts to also expect `*args` in case we add them in the future.